Modules/Transformation/src/BSplineFreeFormTransformationStatistical.cc

```diff
@@ -42,8 +42,8 @@ ::BSplineFreeFormTransformationStatistical()
 BSplineFreeFormTransformationStatistical
 ::BSplineFreeFormTransformationStatistical(const ImageAttributes    &attr,
                                            CPStatus              ****status,
-                                           const Matrix             &mat,
-                                           const Vector             &vec)
+                                           const mirtk::Matrix      &mat,
```

Comment:
Why did you add the namespace here explicitly? This class is itself in the ```mirtk``` namespace. Please revert this and related changes.